src/main/java/mate/academy/rickandmorty/model/CharacterEntity.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.rickandmorty.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "characters")
+public class CharacterEntity {

```

Comment:
No need to have Entity in name of class
```suggestion
public class Character {

```